<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-T9AMV25E" failures="1" tests="9" name="Test" time="56.362" errors="0" timestamp="2024-12-04T22:59:13 IST">
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyAppLaunch" time="5.839"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyRegistration" time="6.949"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyLogOut" time="1.493"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyLogin" time="3.613"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyInitialAccountDetails" time="2.628"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyOpenNewAccount" time="10.372"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyFundsTransfer" time="6.755"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyBillPayment" time="8.281"/>
  <testcase classname="com.parabank.testcases.TC_RegisterCustomerTest" name="verifyTransactionDetailsByAmountAPI" time="4.636">
    <failure type="java.lang.AssertionError" message="expected [16785] but found [16785]">
      <![CDATA[java.lang.AssertionError: expected [16785] but found [16785]
at com.parabank.testcases.TC_RegisterCustomerTest.verifyTransactionDetailsByAmountAPI(TC_RegisterCustomerTest.java:205)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyTransactionDetailsByAmountAPI -->
</testsuite> <!-- Test -->
